lib/features/auth/screens/login_screen.dart

```diff
@@ -50,186 +81,174 @@ class _LoginScreenState extends State<LoginScreen> {
+                Container(
+                  width: double.infinity,
+                  height: size.height * 0.33,
+                  alignment: Alignment.center,
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.center,
+                    mainAxisSize: MainAxisSize.min,
+                    children: [
+                      Text(
+                        'Welcome',
+                        style:
+                            Theme.of(context).textTheme.headlineLarge?.copyWith(
+                                  color: AppTheme.givt4KidsBlue,
+                                  fontWeight: FontWeight.bold,
+                                ),
+                      ),
+                      const SizedBox(height: 5),
+                      Text(
+                        'Ask your parent(s) to sign in\nwith their Givt account',
+                        textAlign: TextAlign.center,
+                        style: Theme.of(context).textTheme.titleLarge?.copyWith(
+                              color: AppTheme.defaultTextColor,
+                            ),
+                      ),
+                    ],
                   ),
                 ),
                 Column(
+                  mainAxisSize: MainAxisSize.min,
                   crossAxisAlignment: CrossAxisAlignment.start,
                   children: [
                     Text(
                       "Email",
-                      style: TextStyle(
-                        color: Color(0xFF3B3240),
-                        fontSize: 15,
-                      ),
-                    ),
-                    SizedBox(
-                      height: 5,
+                      style: Theme.of(context).textTheme.titleMedium?.copyWith(
+                            color: (state is InputFieldErrorState &&
+                                    state.emailErrorMessage.isNotEmpty)
+                                ? AppTheme.givt4KidsRedAlt
+                                : AppTheme.defaultTextColor,
+                            fontWeight: FontWeight.bold,
+                          ),
                     ),
-                    Container(
-                      decoration: BoxDecoration(
-                        borderRadius: BorderRadius.circular(10),
-                        color: Color(0xFFBFDBFC),
-                      ),
-                      padding: EdgeInsets.symmetric(
-                        horizontal: 10,
-                        vertical: 5,
-                      ),
-                      child: TextField(
-                        key: ValueKey("email"),
-                        decoration: InputDecoration(
-                          border: InputBorder.none,
-                          errorText: state is InputFieldErrorState
-                              ? state.emailErrorMessage
-                              : null,
+                    const SizedBox(height: 5),
+                    TextField(
```

Comment:
Should we make it a component with the right borders/style so its reusable? :) (below is the same kind of style)

Reply:
actually, we want to both redesign (as discussed with Elmar) and reimplement (as discussed with Alex) login screen with bloc + formz, so I would suggest to leave this suggestion for upcoming rework?